.eslintrc

```diff
@@ -108,8 +109,7 @@
         "space-unary-ops": 2,
         "spaced-comment": [2, "always", { exceptions: ["-"]}],
         "padded-blocks": [2, "never"],
-        "no-multiple-empty-lines": [2, {max: 2}],
-        "lines-around-comment": [2, { "beforeBlockComment": true, "beforeLineComment": true, "allowBlockStart": true }]
```

Comment:
This rule bugged me, so I took it out. If you feel strongly about it, let me know.